docs/source/modref.rst

```diff
@@ -84,8 +86,10 @@ Miscellaneous commands
    api.check_dates
    api.configuration
    api.export_archive
+   api.export_archive_ora
    api.export_to_figshare
    api.no_annex
+   api.shell_completion
```

Comment:
I know that would be much to ask but since it seems to be "common" to forget to include them in modref, a unittest could be added to https://github.com/datalad/datalad/blob/HEAD/datalad/tests/test_api.py#L81 or around to alert when interface is not present in modref.rst if it would find it present?

not sure though if wouldn't interfer with the cases of extensions.  so consider as a really weak wishlist

Reply:
Thanks for pointing out that test, I wasn't aware of it. So the added test would need to parse `modref.rst`?

With extensions though - if they are listed in interface groups, then I suppose the outcome would depend on what is installed in test environment (which doesn't have to be CI). Although taking a random virtualenv and quickly checking, that might not be the case.

I guess that could be done, question is whether it should be done ;)